storm/datadog_checks/storm/storm.py

```diff
@@ -209,8 +214,7 @@ def from_string(cls, version_string):
             :return: Storm Version
             :rtype: StormCheck.StormVersion
             """
-            parts = version_string.split(".")
-
+            version_string.split(".")
```

Comment:
looks like this should be simply
```suggestion
            parts = version_string.split(".")
            patch_parts = parts[2].split("-")
            return cls(parts[0], parts[1], patch_parts[0], patch_parts[1])
```